src/tests/ov_helpers/ov_models/include/ov_models/preprocess/preprocess_builders.hpp

```diff
@@ -476,6 +479,33 @@ inline std::vector<preprocess_func> generic_preprocess_functions() {
     };
 }
 
+inline std::shared_ptr<Model> cvt_color_rgb_to_bgr() {
+    using namespace ov::preprocess;
+    auto function = create_preprocess_1input(element::f32, PartialShape{1, 20, 30, 3});
+    auto p = PrePostProcessor(function);
+    p.output().model().set_layout("NHWC").set_color_format(ColorFormat::RGB);
+    p.output().postprocess().convert_color(ColorFormat::BGR);
+    function = p.build();
+    return function;
+}
+
+inline std::shared_ptr<Model> cvt_color_bgr_to_rgb() {
+    using namespace ov::preprocess;
+    auto function = create_preprocess_1input(element::f32, PartialShape{1, 20, 30, 3});
+    auto p = PrePostProcessor(function);
+    p.output().model().set_layout("NHWC").set_color_format(ColorFormat::BGR);
+    p.output().postprocess().convert_color(ColorFormat::RGB);
+    function = p.build();
+    return function;
+}
+
+inline std::vector<postprocess_func> generic_postprocess_functions() {
+    return std::vector<postprocess_func>{
+        postprocess_func(cvt_color_rgb_to_bgr, "convert_color_rgb_to_bgr", 0.01f),
```

Comment:
I suppose the difference should be smaller, because we just shuffle values and don't perform any computations.

Reply:
agree, the difference almost is zero, I dumped the result and the output data was shuffled(RGB->BGR) without any value itself modification.